src/extension/providers/debug_adapter_logger_factory.ts

```diff
@@ -1,37 +1,23 @@
 import { DebugAdapterTracker, DebugAdapterTrackerFactory, DebugSession } from "vscode";
-import { DebuggerType, LogCategory } from "../../shared/enums";
-import { IAmDisposable, Logger } from "../../shared/interfaces";
-import { captureLogs, CategoryLogger, EmittingLogger } from "../../shared/logging";
-import { config } from "../config";
-import { insertSessionName } from "../utils";
-import { getLogHeader } from "../utils/log";
+import { LogCategory } from "../../shared/enums";
+import { Logger } from "../../shared/interfaces";
+import { CategoryLogger } from "../../shared/logging";
 
 export class DartDebugAdapterLoggerFactory implements DebugAdapterTrackerFactory {
-	constructor(private readonly logger: EmittingLogger) { }
```

Comment:
Same here and below, I think you've reverted this PR to before my changes so it's trying to undo them.